From 63b918db8c0dcf85aee3f9761293942fa083d7a6 Mon Sep 17 00:00:00 2001
From: Andre Miranda <andreldm@xfce.org>
Date: Sat, 23 Dec 2017 21:45:09 -0300
Subject: Allow compilation against libxfce4ui 4.12 (Bug #14100)

---
 thunar-volman-settings/main.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/thunar-volman-settings/main.c b/thunar-volman-settings/main.c
index d6ce5c5..4121166 100644
--- a/thunar-volman-settings/main.c
+++ b/thunar-volman-settings/main.c
@@ -105,7 +105,12 @@ main (int    argc,
       plug_child = list->data;
       g_list_free (list);
 
+#if LIBXFCE4UI_CHECK_VERSION (4, 13, 0)
       xfce_widget_reparent (plug_child, plug);
+#else
+      /* fallback to deprecated function */
+      gtk_widget_reparent (plug_child, plug);
+#endif
 
       gtk_main ();
       gtk_widget_destroy (plug);
-- 
cgit v1.1

